docs/StardustDocs/topics/overview.md

```diff
@@ -11,25 +11,31 @@ This documentation is written in such a way that it could be read sequentially a
 
 Data frame is an abstraction for working with structured data. Essentially it’s a 2-dimensional table with labeled columns of potentially different types. You can think of it like a spreadsheet or SQL table, or a dictionary of series objects.
 
-The handiness of this abstraction is not in the table itself but in a set of operations defined on it. `Kotlin Dataframe` library is an idiomatic Kotlin DSL defining such operations. The process of working with data frame is often called *data wrangling* which is the process of transforming and mapping data from one "raw" data form into another format that is more appropriate for analytics and visualization. The goal of data wrangling is to assure quality and useful data.
+The handiness of this abstraction is not in the table itself but in a set of operations defined on it. 
+Kotlin Dataframe Library is an idiomatic Kotlin DSL defining such operations. 
+The process of working with data frame is often called *data wrangling* which 
+is the process of transforming and mapping data from one "raw" data form into another format 
+that is more appropriate for analytics and visualization. 
+The goal of data wrangling is to assure quality and useful data.
 
 ## Main Features and Concepts
 
-* [**Hierarchical**](hierarchical.md) — [`DataFrame`](DataFrame.md)  is able to read and present data from different sources including not only plain **CSV** but also **JSON**. That’s why it has been designed hierarchical and allows nesting of columns and cells.
+* [**Hierarchical**](hierarchical.md) — Kotlin DataFrame Library provides an ability to read and present data from different sources including not only plain **CSV** but also **JSON**. That’s why it has been designed hierarchical and allows nesting of columns and cells.
```

Comment:
L -> l

Reply:
everywhere in the file :)
"The Kotlin DataFrame library"

Reply:
Really? I supposed that L just makes it Cooler!

Reply:
haha, well unless we put library in the actual name ;P But we just call it "Kotlin DataFrame"

Reply:
Kotlin DataFrame THE Library